mmdet/utils/misc.py

```diff
@@ -74,3 +75,31 @@ def update(cfg, src_str, dst_str):
 
     update(cfg.data, cfg.data_root, dst_root)
     cfg.data_root = dst_root
+
+
+def get_test_pipeline_cfg(cfg: Union[str, ConfigDict]) -> ConfigDict:
+    """Get the test dataset pipeline from entire config.
+
+    Args:
+        cfg (str or :obj:`ConfigDict`): the entire config. Can be a config
+            file or a ``ConfigDict``.
+
+    Returns:
+        :obj:`ConfigDict`: the config of test dataset.
+    """
+    if isinstance(cfg, str):
```

Comment:
Have you verified this configuration in `configs/yolox/yolox_s_8xb8-300e_coco.py`?

Reply:
Yes. I have tested this config using image_demo.py. The code works well.